resources/language/resource.language.de_de/strings.po

```diff
@@ -190,8 +190,8 @@ msgid "Uncompressing image..."
 msgstr "Entpacke das Image..."
 
 msgctxt "#30046"
-msgid "This may take several minutes. ({mins:d}:{secs:02d})"
-msgstr "Dies kann einige Minuten in Anspruch nehmen. ({mins:d}:{secs:02d})"
+msgid "This may take several minutes."
```

Comment:
The reason we show time here is to make sure the user knows things are progressing.
Otherwise he may think things are stalling and may cancel (which is worse as the thread may still be continuing).

So I prefer we update something every second just in case.

Reply:
Maybe the percentage as floating point?

Reply:
In my tests on an RPi3 the updates of the progress bar seemed fast enough to me. I think even on slower devices this should be fine.

Reply:
During download the progress bar might increase even less frequently, depending on your network connection.

Reply:
Correct, during download we have the same problem.

And the main problem is that we started off with a 600MB file, but now newer ZIP files are 2x and even 2.5x that size. And given that older RPi devices have 1GB or less memory (resulting in smaller file caches) it will take proportionally longer than that if file sizes increase.

So I'd rather have progress be updated with a counter or time advancing, also during download. 
(In fact we could give an indication of the remaining time based on current performance)

Reply:
> (In fact we could give an indication of the remaining time based on current performance)

That's a good idea. I implemented a rather basic version.